prow/config.yaml

```diff
@@ -6824,7 +6824,52 @@ periodics:
         secretName: ssh-key-secret
     - name: var-lib-docker
       emptyDir: {}
-
+- name: ci-ingress-gce-e2e
+  agent: kubernetes
+  interval: 60m
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20180108-bf63630a9-master
+      args:
+      - "--repo=k8s.io/ingress-gce=$(PULL_REFS)"
+      - "--root=/go/src/"
+      - "--upload=gs://kubernetes-jenkins/pr-logs"
```

Comment:
just drop the flag and use the default `--upload` value here (which is `gs://kubernetes-jenkins/logs`)

Reply:
Done.